src/test/groovy/graphql/schema/idl/SchemaTypeCheckerTest.groovy

```diff
@@ -32,7 +32,7 @@ import static java.lang.String.format
 class SchemaTypeCheckerTest extends Specification {
 
     static TypeDefinitionRegistry parseSDL(String spec) {
-        new SchemaParser().parse(spec)
+        new SchemaParser().parse(spec).readOnly()
```

Comment:
This is more real - in the production code the registry is immutable read only mode